lnbits/core/tasks.py

```diff
@@ -20,24 +25,13 @@
 watchdog: Optional[asyncio.Task] = None
 
 
-async def register_killswitch():
+def register_killswitch():
     """
     Registers a killswitch which will check lnbits-status repository
     for a signal from LNbits and will switch to VoidWallet if the killswitch is triggered.
     """
     logger.debug("Starting killswitch task")
-    global killswitch
-    killswitch = asyncio.create_task(killswitch_task())
-
```

Comment:
- any reason for `killswitch: Optional[asyncio.Task] = None` to still be present?
- see: https://github.com/lnbits/lnbits/blob/2928209659950f27cadd4701f5a10c84a75efe2b/lnbits/core/tasks.py#L24